packages/http/src/__tests__/fixtures/index.ts

```diff
@@ -246,9 +222,6 @@ export const httpOperations: IHttpOperation[] = [
     id: 'todos',
     method: 'post',
     path: '/todos',
-    servers: [],
```

Comment:
Does this mean that servers can now be `undefined`?

Reply:
That is correct